kolibri/plugins/html5_app_renderer/content_types.json

```diff
@@ -5,6 +5,12 @@
     "extensions": [
         "zip"
       ]
+    },
+    {
+    "name": "h5p",
```

Comment:
Does it need to be a different kind to `html5`? Seems simpler not to proliferate content kinds?